x-pack/plugins/fleet/public/applications/fleet/sections/agents/services/agent_status.tsx

```diff
@@ -33,20 +20,24 @@ export const AGENT_STATUSES: SimplifiedAgentStatus[] = [
   'unenrolled',
 ];
 
-export function getColorForAgentStatus(agentStatus: SimplifiedAgentStatus): string {
+export function getColorForAgentStatus(
+  agentStatus: SimplifiedAgentStatus,
+  euiTheme: EuiThemeComputed<{}>
+): string {
+  // using variables as mentioned here https://eui.elastic.co/#/theming/colors/values
   switch (agentStatus) {
     case 'healthy':
-      return colorToHexMap.success;
+      return euiTheme.colors.success;
```

Comment:
Let's use semantic tokens here, specifically the background ones:

- `success` => `backgroundFilledSuccess`
- `warning` => `backgroundFilledWarning`
- `primary` => `backgroundFilledPrimary`
- `disabled` => `backgroundBaseDisabled`

Reply:
@mgadewoll Thanks for taking a look at this. Can you please review the latest change? I've updated the PR description with up-to-date screenshots.

Reply:
For added context, here is where the colours of the agent status badges are defined:
https://github.com/elastic/kibana/blob/c92899e142c3edf8318d0a7a13c15ff7cfb3a0f4/x-pack/plugins/fleet/public/applications/fleet/sections/agents/components/agent_health.tsx#L49-L107

As part of this PR, we should ensure that the colours in the table header are aligned with those.